src/warnings.cpp

```diff
@@ -39,15 +39,19 @@ void SetfLargeWorkInvalidChainFound(bool flag)
 
 std::string GetWarnings(const std::string& strFor)
 {
+    static const char* PRE_RELEASE_WARNING = "This is a pre-release test build - use at your own risk - do not use for mining or merchant applications";
```

Comment:
Nit:  You could make this `constexpr`, as in:

    static constexpr const char PRE_RELEASE_WARNING[] = "...";